ckanext/dcat/profiles.py

```diff
@@ -459,7 +457,8 @@ def _access_rights(self, subject, predicate):
             if isinstance(obj, BNode) and self._object(obj, RDF.type) == DCT.RightsStatement:
                 result = self._object_value(obj, RDFS.label)
             elif isinstance(obj, Literal) or isinstance(obj, URIRef):
-                result = six.text_type(obj)
+                # unicode_safe not include Literal or URIRef
+                result = str(obj)
```

Comment:
Not sure on this, `unicode_safe` do not covers this type of object.

Reply:
There is no specific problem. Most PRs are making this change and I assume we want to leave `six.text_type`
I will back to `six` and continue with this PR @amercader 
I just moved to draft